AppRentCar/forms.py

```diff
@@ -9,4 +9,9 @@ class Meta:
         fields = '__all__'
 
 
+class RentForm(forms.ModelForm):
+    class Meta:
+        model = Rent
+        exclude = ('amount',)
```

Comment:
tutaj też usera nie chcemy wybierać, powinno się przypisać od razu do zalogowanego

Reply:
dobra postaram się to rozwiązać, mam pewien trop

Reply:
Sposobów na pewno da się parę rozkminić, wypróbuj ten